webapp-django/crashstats/crashstats/templates/crashstats/home.html

```diff
@@ -61,7 +61,7 @@ <h2 id="homepage-heading">{{ product }} {% if version %}{{ version }}{% endif %}
                             </a>
                         </li>
                         <li>
-                            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;versions={{ version }}&amp;crash_type=plugin&amp;days={{days}}">
+                            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;versions={{ version }}&amp;process_type=plugin&amp;days={{days}}">
```

Comment:
I totally agree with you, but I don't have time to build this before the work week, so I filed a bug about it: https://bugzilla.mozilla.org/show_bug.cgi?id=1229832